docs/source/nlp/tutorials/quickstarts/plot_multi_label_classification.py

```diff
@@ -0,0 +1,228 @@
+class_names = label_cols.columns.to_list()
+dataset = TextData(data['originalText'], label=label_cols.to_numpy().astype(int),
+                   task_type='text_classification', metadata=data[metadata_cols], categorical_metadata=[])
+
+# %%
+# Calculating Properties
+# ----------------------
+#
+# Some of deepchecks' checks use properties of the text samples for various calculations. Deepcheck has a wide variety
+# of such properties, some simple and some that rely on external models and are more heavy to run. In order for
+# deepchecks' checks to be able to access the properties, they must be stored within the TextData object.
+
+# properties can be either calculated directly by Deepchecks or
+# imported from other sources in an :ref:`appropriate format <nlp__properties_guide>.
+
+# device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+# dataset.calculate_default_properties(include_long_calculation_properties=True, device=device)
+
+properties = just_dance_comment_analysis.load_properties(as_train_test=False)
+dataset.set_properties(properties, categorical_properties=['Language'])
+dataset.properties.head(2)
+
+# %%
+# Running the deepchecks default suites
+# =====================================
+#
+# Data Integrity
+# --------------
+# We will start by doing preliminary integrity check to validate the text formatting. It is recommended to do this step
+# before your train and test/validation splits and model training as it may imply additional data
+# engineering is required.
+#
+# We'll do that using the data_integrity pre-built suite. Note that we are limiting the number of samples to 1000
+# in order to get quick high level overview of potential issues.
+
+from deepchecks.nlp.suites import data_integrity
+
+data_integrity_suite = data_integrity(n_samples=1000)
+data_integrity_suite.run(dataset, model_classes=class_names)
+
+# %%
+# Integrity #1: Unknown Tokens
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# First up (in the “Didn’t Pass” tab) we see that the Unknown Tokens check
+# has returned a problem.
+#
+# Looking at the result, we can see that it assumed (by default) that
+# we’re going to use the bert-base-uncased tokenizer for our NLP model,
+# and that if that’s the case there are many words in the dataset that
+# contain characters (specifically here emojis) that are
+# unrecognized by the tokenizer. This is an important insight, as bert
+# tokenizers are very common.
+#
+# Integrity #2: Conflicting Labels
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Looking at the Conflicting Labels check result (in the “Didn’t Pass” tab) we can
+# see that there are 2 occurrences of duplicate samples that have different labels.
+# This may suggest a more severe labeling error in the dataset which we would want to explore further.
+#
+
+# %%
+# Train Test Validation
+# ---------------------
+# The next suite serves to validate our split and compare the two dataset. This suite is useful for when you already
```

Comment:
same comment as in other quickstart (mention additional use cases of this not only in the directly train/test context)